### PR TITLE
truthfy:0.1.0

### PR DESCRIPTION
**Pull request by ThumusLive**
I am submitting
- [x] a new package
- [ ] an update for a package

Create truth table automatically from math blocks. It can resolve truth table, [see this](https://github.com/ThumusLive/truthfy)

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

File: packages/preview/truthfy/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+repository = "https://github.com/ThumusLive/truthfy"
+homepage = "https://github.com/ThumusLive/truthfy"
+keywords = [
+    "truth",
+    "table",
+]
+exclude = []
+
+[utpm]
+version = "1"
```
Comment (laurmaedje) [inline packages/preview/truthfy/0.1.0/typst.toml:17]: What is the purpose of this key? I couldn't find any explanation on https://github.com/ThumusLive/utpm. Why does this need to be part of the package manifest?

Comment (ThumusLive) [reply packages/preview/truthfy/0.1.0/typst.toml:17]: I'm working on it, I'm gonna change things in the future so I need to track the version (but it's not published yet)
I'll delete it

**Comment by ThumusLive** [comment]
Do you need only one commit or that's fine?

**Comment by laurmaedje** [comment]
It's fine, I squash and merge anyway.

File: packages/preview/truthfy/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# truthfy
+Make an empty or filled truth table in Typst
+
+# Functions
+
+```
+generate_empty(info: array[math_block], data: array[str]): table
```
Comment (laurmaedje) [inline packages/preview/truthfy/0.1.0/README.md:7]: Just wanted to note that the idiomatic Typst case would be `generate-empty` (i.e. kebab-case). It's up to you whether you want to change it to that or not though.

Comment (ThumusLive) [reply packages/preview/truthfy/0.1.0/README.md]: Done!

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**